values.yaml

```diff
@@ -1277,6 +1277,12 @@ serverTelemetry:
     # Timeout for Prometheus scrapes
     scrapeTimeout: 10s
 
+    # tlsConfig used for connecting to the Vault API
+    tlsConfig: {}
+
+    # bearerTokenfile used for authentication to the Vault metrics API
+    bearerTokenFile: ""
```

Comment:
There is a comment up above that needs a bit of updating:

>  ```
> # Enable support for the Prometheus Operator. Currently, this chart does not support
>  # authenticating to Vault's metrics endpoint, so the following `telemetry{}` must be included
>  # in the `listener "tcp"{}` stanza
>  #  telemetry {
>  #    unauthenticated_metrics_access = "true"
>  #  }
> ```

Maybe something along the lines of:

>  ```
> # Enable support for the Prometheus Operator. If bearerTokenFile is not set for authenticating
>  # to Vault's metrics endpoint, the following Vault server `telemetry{}` config must be included
>  # in the `listener "tcp"{}` stanza
>  #  telemetry {
>  #    unauthenticated_metrics_access = "true"
>  #  }
> ```

Reply:
011f82c

Reply:
wait my lazy-vim did a lot of changes, gonna revert them.

Reply:
There we go 71d6325